tests/unit/index.ts

```diff
@@ -8,7 +8,8 @@ import '../../src/reducers/Profile.test'
 import '../../src/reducers/SignIn.test'
 import '../../src/reducers/SignUp.test'
 import '../../src/reducers/User.test'
+import '../../src/sagas/fetchSaga.test'
```

Comment:
Should be a captial F `import '../../src/sagas/FetchSaga.test'`. test is failing to import on my machine.